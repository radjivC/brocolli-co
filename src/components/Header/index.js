import React from 'react'

const Header = () => {
  return (
    <div className="hero-head">
      <nav className="navbar">
        <div className="container">
          <div className="navbar-brand">
            <a className="navbar-item">
              Brocolli and Co
            </a>
          </div>
        </div>
      </nav>
    </div>
  )
}

export default Header
