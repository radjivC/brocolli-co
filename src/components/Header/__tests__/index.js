import React from 'react'
import renderer from 'react-test-renderer'
import Header from '../'

describe('Header', () => {
  it('can render without crashing', () => {
    const component = renderer.create(
      <Header />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
