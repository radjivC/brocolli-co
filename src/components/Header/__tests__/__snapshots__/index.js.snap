// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header can render without crashing 1`] = `
<div
  className="hero-head"
>
  <nav
    className="navbar"
  >
    <div
      className="container"
    >
      <div
        className="navbar-brand"
      >
        <a
          className="navbar-item"
        >
          Brocoli and Co
        </a>
      </div>
    </div>
  </nav>
</div>
`;
